src/main/scala/fr/polytechnique/cmap/cnam/util/RichDataFrame.scala

```diff
@@ -55,6 +55,21 @@ class RichDataFrame(dataFrame: DataFrame) {
       .parquet(path)
   }
 
+  def writeOrc(path: String, mode: String = "errorIfExists"): Unit = {
+    dataFrame.write
+      .mode(saveMode(mode))
+      .orc(path)
+  }
+
+  def write(path: String, mode: String = "errorIfExists", format: String = "parquet"): Unit = {
+    format match {
+      case "orc" => writeOrc(path, mode)
+      case "csv" => writeCSV(path, mode)
+      case _ => writeParquet(path, mode)
```

Comment:
It might be better to have a `"parquet"` case in addition to a default case `_` to avoid implicit behaviour. The default case would either raise a warning such as `format not understood, using 'parquet' as default` or an error.

Reply:
Yes it's better